package.json

```diff
@@ -84,8 +84,8 @@
     "lint": "tslint --project . && npm run prettier-check",
     "postpack": "rm -f oclif.manifest.json",
     "prepack": "rm -rf lib tsconfig.tsbuildinfo && tsc -b && oclif-dev manifest && oclif-dev readme",
-    "prettier": "prettier --write '**/*.{ts,tsx,json}' --ignore-path .gitignore",
-    "prettier-check": "prettier --check '**/*.{ts,tsx,json}' --ignore-path .gitignore",
+    "prettier": "prettier --write \"**/*.{ts,tsx,json}\" --ignore-path .gitignore",
+    "prettier-check": "prettier --check \"**/*.{ts,tsx,json}\" --ignore-path .gitignore",
     "test": "jest --coverage --detectOpenHandles ./test",
```

Comment:
Single quote is not working on windows and `yarn lint`, `yarn prettier` and `yarn prettier-check` throws an error.
Replacing them to double quote is working for both unix and windows.